### ISSUE TITLE
Option to pass SIGINT to the command

### ISSUE THREAD

**Issue by tennox** [go-task/task#810]
I have a task `dev` that runs the app's dev server. At some point I will want to stop it via CTRL-C - which is normally handled by the command itself by exiting without error:
![Screenshot from 2022-07-15 11-34-39](https://user-images.githubusercontent.com/2084639/179207087-4650337b-b710-4f17-9a83-029c458c8fdd.png)

When running the task via `task dev` though, `task` will treat the SIGINT as a kind of failure though:
![image](https://user-images.githubusercontent.com/2084639/179207287-53c9274a-c3be-4056-b719-412c8bf37b5a.png)

I'm thinking of an option `forward_signals` - and I guess it's more appropriate on a specific command than on the task, as some commands inside a task might handle SIGINT and some don't:
```yaml
cmds:
  - cmd: meteor run
    forward_signals: true
```

**Comment by tennox** [comment]
Fair enough,
> Task version: 3.12.1
But with `v3.14.0 (h1:/00rpwoB+lo9ztU7MPa9aCDywZvrXaxXNR2BxvyEhwE=)` I still get
![image](https://user-images.githubusercontent.com/2084639/179357958-7ffce872-d0a1-4b26-beea-f44188e6c84a.png)

**Comment by ghostsquad** [comment]
If you had to interrupt it, it didn't complete successfully. The exit code of the process is what task is looking at (afaik).

**Comment by andreynering** [comment]
Hi @tennox,

When you send a SIGINT signal to Task, you're actually sending that signal to the whole process group, which means that all processes run by Task also receive the signal.

The message is saying it is working as expected, the called process exited after receiving the signal, with the said error code.

Let me know if you have more questions.

**Comment by tennox** [comment]
Yeah, you're right.

So the issue seems to be that `task` returns exit code `1` but the command returns `130`
```fish
task: Failed to run task "exit": exit status 130
$ echo $status
1
```

Since
> 130 exit status is considered as a death by SIGINT.

Maybe there can be an option to return same exit code (can imagine other situations where this is useful)?

**Comment by ghostsquad** [comment]
@tennox this is already available with `--exit-code` flag via #755

**Comment by tennox** [comment]
Oh. Thanks 🙈
